ballerina/constants.bal

```diff
@@ -63,3 +63,17 @@ public enum NamespaceUUID {
     NAME_SPACE_X500 = "6ba7b814-9dad-11d1-80b4-00c04fd430c8",
     NAME_SPACE_NIL = NIL_UUID
 }
+
+# The duration between the Gregorian time (1582-10-15 0:0:0) and UTC start time (1970-01-01 0:0:0) in seconds.
+const decimal gregorianTimeInSeconds = 12219292800;
```

Comment:
```suggestion
const decimal GREGORIAN_TIME_IN_SECONDS = 12219292800;
```

Reply:
Fixed in [9f00e65](https://github.com/ballerina-platform/module-ballerina-uuid/pull/376/commits/9f00e655adb4e181800a418926b0067c0512d2fb)